### ISSUE TITLE
SGD+Nesterov optimizer

### ISSUE THREAD

**Issue by albarji** [albarji/neurowriter#13]
Recent results show that while adaptive optimization methods do obtain better minima in the trainining loss function, they are more prone to overfitting, specially in network with more parameters than training data, which might well be the case here.

To avoid this it would be useful to SGD and SGD+Nesterov as optimizers in the hypertuning procedure.
* https://keras.io/optimizers/
* https://arxiv.org/abs/1705.08292
